test/es6/ES6Function_bugs.js

```diff
@@ -28,7 +28,7 @@ var tests = [
                     get: function () { }
                     });
             }
-            assert.doesNotThrow(function () { g('length') }, "assertion failure on defineProperty 'length' with getter after sealing a function object");
+            assert.throws(function () { g('length') }, TypeError, "Cannot redefine non-configurable property 'length'");
             assert.throws(function () { g('arguments') }, TypeError, "Cannot redefine non-configurable property 'arguments'");

```

Comment:
This test was testing incorrect behavior see: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/seal